libs/client/use-cases/dashboard/shared/ui/src/lib/header/header.component.html

```diff
@@ -1 +1,31 @@
-<p>header works!</p>
+<head><script src="https://unpkg.com/feather-icons"></script></head>
+<header>
+    <div class="header__item">
+        <button class="header__hamburger" (click)="onClickMenuIcon()">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line></svg>
```

Comment:
?? 
I would prefer you to use ready-made icons from taiga ui to start with
an example of use you can find in the view header component

Reply:
https://taiga-ui.dev/icons